tests/h/h_api/schema/schema_file_test.py

```diff
@@ -0,0 +1,53 @@
+import os
+
+import pytest
+from jsonschema import Draft7Validator
+
+from h.h_api.schema import Schema
+
+
+def schema_file_names():
+    for path, _, names in os.walk(Schema.BASE_DIR):
+        for name in names:
+            full_path = os.path.join(path, name)
+            rel_path = os.path.relpath(full_path, Schema.BASE_DIR)
+
+            yield rel_path
+
+
+def schema_with_examples():
```

Comment:
Eugh... I could do with some ideas here. This isn't very nice as it can fail if the schema are invalid outside of the testing framework. This is for a parameterised fixture, but it appears it needs to be done by the time you add the decorator. I was hoping you'd be able to create a fixture that yields out different items or something, but I couldn't see how.